server/service/scripts.go

```diff
@@ -110,6 +121,15 @@ func runScriptSyncEndpoint(ctx context.Context, request interface{}, svc fleet.S
 	}, nil
 }
 
+func (svc *Service) GetScriptIDByName(ctx context.Context, scriptName string, teamID *uint) (uint, error) {
+	// TODO: confirm auth level
```

Comment:
Just checking if that was confirmed? Should we wait for this?

Reply:
I'm guessing we probably don't want to hold for this and can address any small change during freeze?

Reply:
Thanks! I just wanted to flag the issue for discussion and a sanity check. I think this auth level makes sense at this stage of the process, but it is starting to get a bit murky to track due to the overlapping service layer methods between the sync and async flows and all the potential exits that can happen at different levels of auth. I think we can continue the discussion address it during the freeze.